src/effects/effectsmanager.cpp

```diff
@@ -69,10 +71,7 @@ EffectsManager::~EffectsManager() {
 bool alphabetizeEffectManifests(EffectManifestPointer pManifest1,
                                 EffectManifestPointer pManifest2) {
     int dNameComp = QString::localeAwareCompare(pManifest1->displayName(), pManifest2->displayName());
-    int bNameComp = QString::localeAwareCompare(
-                        EffectManifest::backendTypeToTranslatedString(pManifest1->backendType()), 
-                        EffectManifest::backendTypeToTranslatedString(pManifest2->backendType()));
-    // Add an exception for "Built-in" backends, to keep the Built-in effects in the beginning
```

Comment:
Add a comment here: "Sort built-in effects first before external plugins"

Reply:
Let's also add a comment in defs.h where the EffectBackendType enum is declared explaining that the order that the values are declared in the enum class determines how they are sorted in the WEffectSelector and DlgPrefEffects.